e2e/support/helpers/e2e-dashboard-helpers.js

```diff
@@ -236,11 +238,14 @@ export function resizeDashboardCard({ card, x, y }) {
     const resizeHandle = cy.get(".react-resizable-handle");
     resizeHandle
       .trigger("mousedown", { button: 0 })
+      .wait(200)
       .trigger("mousemove", {
         clientX: x,
         clientY: y,
       })
-      .trigger("mouseup", { force: true });
+      .wait(200)
+      .trigger("mouseup", { force: true })
+      .wait(200);
```

Comment:
if we can achieved this without the `wait`s that'd be preferable. slacked you a potential solution.